src/app/components/list/overview/index.html

```diff
@@ -60,6 +60,120 @@
+      isOptional="true"
+    >
+    Specifies the text for the filter's summary item. If not specified, then the filter summary item text defaults to the <sky-code>value</sky-code> of the filter. This property accepts <sky-code>string</sky-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="dismissable"
+      isOptional="true"
+    >
+    Indicates whether users can dismiss the filter's summary item. This property accepts <sky-code>boolean</sky-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="value"
+      isOptional="true"
+    >
+    Specifies the current value of the filter. This property accepts <sky-code>any</sky-code> value.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="defaultValue"
+      isOptional="true"
+    >
+    Specifies the default value of the filter. If the value of the filter is set to the default value, then the filter is not applied.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="filterFunction"
+      isOptional="true"
+    >
+    Specifies the function to apply to determine whether an item is filtered. This property is required when using in memory data provider. Acceptable values: <sky-code>(item: ListItemModel, filter: any) => boolean</sky-code>. This property accepts <sky-code>boolean</sky-code> values.    
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-properties sectionHeading="ListSearchModel properties">
+    <sky-demo-page-property
+      propertyName="searchText"
+      isOptional="true"
+    >
+    Specifies the text to search. This property accepts <sky-code>string</sky-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="functions"
+      isOptional="true"
+    >
+    Specifies an array of functions that should return a <sky-code>boolean</sky-code> value of <sky-code>true</sky-code> when the search is successful. Acceptable values: <sky-code>(data: any, searchText: string) => boolean></sky-code>.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="fieldSelectors"
+      isOptional="true"
+    >
+    Specifies an array of <sky-code>string</sky-code> types which run against search fields. The <sky-code>string</sky-code> values should match the data's <sky-code>field</sky-code> value.
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-properties sectionHeading="ListSortModel properties">
+    <sky-demo-page-property
+      propertyName="available"
+      isOptional="true"
+    >
+    Specifies the list of avilable views to sort. Acceptables values: <sky-code>Array of ListSortLabelModel</sky-code>.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="global"
+      isOptional="true"
+    >
+    Specifies all views to sort. Acceptable values: <sky-code>Array of ListSortLabelModel</sky-code>.
```

Comment:
Same recommendation for the second sentence here as in the previous description. And same question about "views."

Reply:
@Blackbaud-DeniseCisneros, can we resolve this? Or is anything still pending?